src/main/java/com/smartbear/postman/exceptions/PostmanCollectionUnsupportedVersionException.java

```diff
@@ -2,7 +2,10 @@
 
 public class PostmanCollectionUnsupportedVersionException extends Exception {
 
-    public static final String UNSUPPORTED_VERSION_MESSAGE = "Unsupported version of Postman collection was provided";
+    public static final String UNSUPPORTED_VERSION_MESSAGE = "Unsupported version of Postman collection was provided.\n" +
+            "Make sure your collection version is 2.0.0 or 2.1.0 (recommended). " +
```

Comment:
"Make sure your collection version is at least 2.0.0"
This won't specify any recommendation that would have to be updated every time Postman changes something.